docs/buildah-bud.md

```diff
@@ -163,7 +163,7 @@ context.
 
 **--force-rm**
 
-Always remove intermediate containers after a build. Buildah does not currently support caching so this is a NOOP.
+Always remove intermediate containers after a build, even if the build is unsuccessful..
```

Comment:
should line 164 and 272 have bool-value like tls-verify does?

Reply:
Sure! Fixed :)